awesomeConfig.cmake

```diff
@@ -354,13 +354,24 @@ add_custom_target(lgi-check-run ALL
 
 # {{{ Generate some aggregated documentation from lua script
 
-file(MAKE_DIRECTORY ${BUILD_DIR}/script_files/)
+# file(MAKE_DIRECTORY ${BUILD_DIR}/script_files/)
+# file(MAKE_DIRECTORY ${BUILD_DIR}/docs/common/)
+# file(COPY ${SOURCE_DIR}/docs/_parser.lua DESTINATION ${BUILD_DIR}/docs/)
+
+add_custom_target(setup_directories DEPENDS lgi-check-run)
+
+add_custom_command(TARGET setup_directories
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${BUILD_DIR}/script_files/
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${BUILD_DIR}/docs/common/
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${BUILD_DIR}/doc/images/
+        COMMAND ${CMAKE_COMMAND} -E copy ${SOURCE_DIR}/docs/_parser.lua ${BUILD_DIR}/docs/
```

Comment:
Why these new statements? I cannot find anything creating these directories previously.

Reply:
One of them is created in `tests/CMakeLists.txt`. The others by `configure_file`. It fixes #2342. I will add a note in the commit message